examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -634,6 +108,14 @@ constexpr EndpointId kNetworkCommissioningEndpointSecondary = 0xFFFE;
 
 static void InitServer(intptr_t context)
 {
+    ESP_LOGI(TAG, "------------------------Starting App Task---------------------------");
+    CHIP_ERROR error = GetAppTask().StartAppTask();
```

Comment:
This shouldn't be a part of InitServer. Also, why do we need a separate task like App Task? We can reuse the already present/created main task to handle the queue events/messages under M5STACK config option

Reply:
I moved it out of InitServer.  The new App Task will be used to handle the queue events/messages, just like other examples.